packages/core/src/generators/svelte.ts

```diff
@@ -401,7 +426,18 @@ export const componentToSvelte =
           if (name === 'children') {
             return '';
           }
-          return `export let ${name};`;
+
+          let propDeclaration = `export let ${name}`;
+
+          if (propsTypeRef && propsTypeRef !== 'any') {
+            propDeclaration += `: ${
+              isString(propsTypeRef) ? propsTypeRef.split(' |')[0] : propsTypeRef
```

Comment:
Since `propsTypeRef` is guaranteed to be a string here, we should be able to just do:
```suggestion
              propsTypeRef.split(' |')[0]
```